src/Psalm/Internal/Scanner/DocblockParser.php

```diff
@@ -19,7 +19,7 @@
 
 class DocblockParser
 {
-    public static function parse(string $docblock) : ParsedDocblock
+    public static function parse(string $docblock, int $offsetStart) : ParsedDocblock
```

Comment:
Mind adding a docblock specifying whether absolute file offset or relative docblock offset is expected here?

Reply:
Will do!

Reply:
done!